README.md

```diff
@@ -63,8 +63,9 @@ Create a discord user and join the server you want to connect to IRC.
     },
     "ircOptions": { // Optional node-irc options
       "floodProtection": false, // On by default
-      "floodProtectionDelay": 1000 // 500 by default
+      "floodProtectionDelay": 1000, // 500 by default
     },
+    ircNickColor: false, // Gives usernames a color in IRC for better readability (off by default)
```

Comment:
You forgot quotes around `ircNickColor` :smiley:. (`ircNickColor` -> `"ircNickColor"`)